src/app/store/reservedip/types/base.ts

```diff
@@ -0,0 +1,24 @@
+import type { Subnet, SubnetMeta } from "../../subnet/types";
+import type { TimestampedModel } from "../../types/model";
+import type { NetworkInterface, Node, NodeType } from "../../types/node";
+import type { GenericState } from "../../types/state";
```

Comment:
All of these should ideally use absolute imports with `@/`, e.g. `@/app/store/subnet/types`